docs/utilities/idempotency.md

```diff
@@ -204,61 +245,61 @@ The output serializer supports any JSON serializable data, **Python Dataclasses*
 
         Alternatively, you can provide an explicit model as an input to `DataclassSerializer`.
 
-        ```python hl_lines="8 27-29 36 39 48"
+        ```python hl_lines="8 30 38 40 50"
         --8<-- "examples/idempotency/src/working_with_dataclass_explicitly_output_serializer.py"
         ```
 
 === "Any type"
 
-    You can use `CustomDictSerializer` to have full control over the serialization process for any type. It expects two functions:
+    Use `CustomDictSerializer` to have full control over the serialization process for any type. It expects two functions:
 
     * **to_dict**. Function to convert any type to a JSON serializable dictionary before it saves into the persistent storage.
     * **from_dict**. Function to convert from a dictionary retrieved from persistent storage and serialize in its original form.
 
-    ```python hl_lines="8 32 36 40 50 53"
+    ```python hl_lines="9 34 38 42 52 54 64"
     --8<-- "examples/idempotency/src/working_with_idempotent_function_custom_output_serializer.py"
     ```
 
     1. This function does the following <br><br>**1**. Receives the return from `process_order`<br> **2**. Converts to dictionary before it can be saved into the persistent storage.
     2. This function does the following <br><br>**1**. Receives the dictionary saved into the persistent storage <br>**1** Serializes to `OrderOutput` before `@idempotent` returns back to the caller.
     3. This serializer receives both functions so it knows who to call when to serialize to and from dictionary.
 
-#### Batch integration
+### Using in-memory cache
 
-You can can easily integrate with [Batch utility](batch.md){target="_blank"} via context manager. This ensures that you process each record in an idempotent manner, and guard against a [Lambda timeout](#lambda-timeouts) idempotent situation.
+!!! note "In-memory cache is local to each Lambda execution environment."
 
-???+ "Choosing an unique batch record attribute"
-    In this example, we choose `messageId` as our idempotency key since we know it'll be unique.
+You can enable caching with the `use_local_cache` parameter in `IdempotencyConfig`. When enabled, you can adjust cache capacity _(256)_ with `local_cache_max_items`.
```

Comment:
1024?

<img width="745" alt="image" src="https://github.com/aws-powertools/powertools-lambda-python/assets/4295173/1b4fdd28-823e-4113-a556-81cadb4104b3">

Reply:
`LRUDict` implementation is 1024, but we reduce in Idempotency to default to 256 to reduce memory footprint.

https://github.com/aws-powertools/powertools-lambda-python/blob/c77869a89c1ebb87718b78b2027a2f0978512142/aws_lambda_powertools/utilities/idempotency/persistence/base.py#L84